pkg/asset/cluster/azure/azure.go

```diff
@@ -0,0 +1,21 @@
+// Package azure extracts AZURE metadata from install configurations.
+package azure
+
+import (
+	"fmt"
+
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/azure"
+)
+
+// Metadata converts an install configuration to AWS metadata.
```

Comment:
azure

Reply:
yeah, I'll do a broad AWS search in the azure implementations. leave these notes for later :)

Reply:
fixed